docs/user-guide/overview.md

```diff
@@ -1,75 +1,71 @@
-
-Next, let's analyze the key features of GreptimeDB demonstrated by this query example:
-
-- **Unified Storage:** GreptimeDB is the time series database to store and analyze both metrics and [logs](/user-guide/logs/overview.md). The simplified architecture and data consistency enhances the ability to analyze and troubleshoot issues, and can lead to cost savings and improved system performance.
-- **Unique Data Model:** The unique [data model](/user-guide/concepts/data-model.md) with time index and full-text index greatly improves query performance and has stood the test of large data sets. It not only supports metric [insertion](/user-guide/ingest-data/overview.md) and [query](/user-guide/query-data/overview.md), but also provides a very friendly way to [write](/user-guide/logs/write-logs.md) and [query](/user-guide/logs/query-logs.md) logs, as well as handle [vector type data](/user-guide/vectors/vector-type.md).
-- **Range Queries:** GreptimeDB supports [range queries](/user-guide/query-data/sql.md#aggregate-data-by-time-window) to evaluate [expressions](/reference/sql/functions/overview.md) over time, providing insights into metric trends. You can also [continuously aggregate](/user-guide/flow-computation/overview.md) data for further analysis.
-- **SQL and Multiple Protocols:** GreptimeDB uses SQL as the main query language and supports [multiple protocols](/user-guide/protocols/overview.md), which greatly reduces the learning curve and development cost. You can easily migrate from Prometheus or [Influxdb to GreptimeDB](/user-guide/migrate-to-greptimedb/migrate-from-influxdb.md), or just start with GreptimeDB.
-- **JOIN Operations:** The data model of GreptimeDB's time series tables enables it to support [JOIN operations](/reference/sql/join.md) on metrics and logs.
-
-Having understood these features, you can now go directly to exploring the features that interest you, or continue reading the next step in the sequence.
-
-## Next steps
-
-* [Concepts](./concepts/overview.md)
-* [Migrate to GreptimeDB](./migrate-to-greptimedb/migrate-from-influxdb.md)
-* [Ingest Data](./ingest-data/overview.md)
-* [Query Data](./query-data/overview.md)
-* [Manage Data](./manage-data/overview.md)
-* [Integrations](./integrations/overview.md)
-* [Protocols](./protocols/overview.md)
-* [Continuous Aggregation](./flow-computation/overview.md)
-* [Operations](./administration/overview.md)
+
+## Understanding GreptimeDB Concepts
+
+Before diving into GreptimeDB,
+it is recommended to familiarize yourself with its data model, key concepts, and features.
+For a comprehensive overview, 
+refer to the [Concepts Documentation](./concepts/overview.md).
+
+## Ingesting Data Based on Your Use Case
+
+GreptimeDB supports [multiple protocols](./protocols/overview.md) and [various integration tools](./integrations/overview.md) to simplify data ingestion tailored to your requirements.
+
+### For Observability Metrics
```

Comment:
Combine this with logs, and add traces too. Rename the section as "For Observability Data"